chronosphere/enum/enum_test.go

```diff
@@ -66,10 +66,12 @@ func TestEnumValidateError(t *testing.T) {
 
 func TestAllEnumsValidate(t *testing.T) {
 	v1Spec := loadSwaggerSpec(t, "../pkg/configv1/swagger.json")
+	unstableSpec := loadSwaggerSpec(t, "../pkg/configunstable/swagger.json")
 
 	tests := []struct {
 		legacySwaggerName string
 		v1SwaggerName     string
+		swaggerSpec       *openapi2.T
```

Comment:
since we only have stable / unstable, probably simpler to do `unstable bool` here, and have the test runner switch to the unstable spec